src/components/dropdown/textBox/TextBox.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+interface TextBox {
+	message: string;
+	children?: React.ReactNode;
+}
+
+const TextBox = ({ message, children }: TextBox) => {
```

Comment:
```suggestion
import React, {type PropsWithChildren} from 'react';

interface TextBox {
	message: string;
}

const TextBox = ({ message, children }: PropsWithChildren<TextBox>) => {
```

children의 옵셔널 프로퍼티를 사용하지 않아도 이렇게 사용도 가능해용!